src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -35,50 +48,44 @@ def key(self) -> str:
     def kind(self) -> str:
         return "VIEW" if self.view_text is not None else "TABLE"
 
-    def _sql_alter(self, catalog):
+    def sql_alter(self, catalog):
         return (
             f"ALTER {self.kind} {self.key} SET"
             f" TBLPROPERTIES ('upgraded_to' = '{catalog}.{self.database}.{self.name}');"
         )
 
     def _sql_external(self, catalog):
-        # TODO: https://github.com/databricks/ucx/issues/106
-        return (
-            f"CREATE TABLE IF NOT EXISTS {catalog}.{self.database}.{self.name}"
-            f" LIKE {self.key} COPY LOCATION;" + self._sql_alter(catalog)
-        )
+        return f"SYNC TABLE {catalog}.{self.database}.{self.name} FROM {self.key};"
 
     def _sql_managed(self, catalog):
         if not self.is_delta:
             msg = f"{self.key} is not DELTA: {self.table_format}"
             raise ValueError(msg)
-        return (
-            f"CREATE TABLE IF NOT EXISTS {catalog}.{self.database}.{self.name}"
-            f" DEEP CLONE {self.key};" + self._sql_alter(catalog)
-        )
+        return f"CREATE TABLE IF NOT EXISTS {catalog}.{self.database}.{self.name} DEEP CLONE {self.key};"
 
     def _sql_view(self, catalog):
         return f"CREATE VIEW IF NOT EXISTS {catalog}.{self.database}.{self.name} AS {self.view_text};"
 
     def uc_create_sql(self, catalog):
         if self.kind == "VIEW":
             return self._sql_view(catalog)
-        elif self.location is not None:
+        elif self.object_type == "EXTERNAL":
             return self._sql_external(catalog)
         else:
             return self._sql_managed(catalog)
 
 
 class TablesCrawler(CrawlerBase):
-    def __init__(self, backend: SqlBackend, schema):
+    def __init__(self, backend: SqlBackend, cfg: WorkspaceConfig):
```

Comment:
don't depend on the whole `WorkspaceConfig`, depend only on the properties you need. e.g `inventory_database`, `database_to_catalog_mapping`, and `default_catalog`. add defaults to those properties in constructor as well